lib/backburner.js

```diff
@@ -516,8 +516,8 @@ function updateLaterTimer(backburner, executeAt, wait) {
       clearTimeout(backburner._laterTimer);
 
       if (backburner._laterTimerExpiresAt < n) { // If timer was never triggered
-        // Calculate the left-over wait-time
-        wait = Math.max(0, executeAt - n);
+        // execute it immediately
+        wait = 0;
```

Comment:
would it be better to just call executeTimers(backburner) and return right away rather than setting wait to 0?